### PR TITLE
docs: Make "Install guest kernel images" arch-sensitive

### PR DESCRIPTION
**Pull request by nitkon**
Make "Install guest kernel images" in Developer-Guide
arch specific.

Fixes: #140

Signed-off-by: Nitesh Konkar <niteshkonkar@in.ibm.com>

### CODE REVIEW

File: Developer-Guide.md [hunk 1]
```diff
@@ -277,10 +280,10 @@ $ tar -xf ${kernel_tar_file}
 $ mv .config "linux-${kernel_version}"
 $ pushd "linux-${kernel_version}"
 $ curl -L https://raw.githubusercontent.com/kata-containers/packaging/master/kernel/patches/0001-NO-UPSTREAM-9P-always-use-cached-inode-to-fill-in-v9.patch | patch -p1
-$ make ARCH=${kernel_arch} -j$(nproc)
+$ make ARCH=${kernel_dir} -j$(nproc)
 $ kata_kernel_dir="/usr/share/kata-containers"
 $ kata_vmlinuz="${kata_kernel_dir}/kata-vmlinuz-${kernel_version}.container"
-$ sudo install -o root -g root -m 0755 -D "$(realpath arch/${kernel_arch}/boot/bzImage)" "${kata_vmlinuz}"
+$ if [ $kernel_arch == ppc64le ]; then sudo install -o root -g root -m 0755 -D "$(realpath ./vmlinux)" "${kata_vmlinuz}"; else sudo install -o root -g root -m 0755 -D "$(realpath arch/${kernel_arch}/boot/bzImage)" "${kata_vmlinuz}"; fi
```
Comment (jodh-intel) [inline Developer-Guide.md]: I'd be tempted to simplify this further by removing the `if` and only defining the `install` line once. Something like:

```
$ ["$kernel_arch = ppc64le ] && kernel_file="$(realpath ./vmlinux)" || kernel_file="$(realpath arch/${kernel_arch}/boot/bzImage)"
$ sudo install -o root -g root -m 0755 -D "$kernel_file" "${kata_vmlinuz}"
```

File: Developer-Guide.md [hunk 2]
```diff
@@ -265,10 +265,13 @@ As a prerequisite, you need to install `libelf-dev` and `bc`. Otherwise, you
 will not be able to build the kernel from sources.
 
 ```
-$ kernel_arch="$(arch)"
+$ go get github.com/kata-containers/tests
+$ cd $GOPATH/src/github.com/kata-containers/tests/.ci
+$ kernel_arch="$(./kata-arch.sh --golang)"
+$ kernel_dir="$(./kata-arch.sh --kernel)"
```
Comment (jodh-intel) [inline Developer-Guide.md]: \o/ :smile:

Comment (nitkon) [reply Developer-Guide.md]: @jodh-intel : Thanks for your work regarding the script. I have fixed my patch based on your above comment.

**Comment by jodh-intel** [comment]
lgtm

**Comment by jodh-intel** [comment]
CI failing as #150 hasn't landed yet.

**Comment by nitkon** [comment]
@sboeuf @grahamwhaley

**Comment by nitkon** [comment]
Travis is passing now post  #150 merger.

**Comment by jodh-intel** [comment]
Hi @klynnrif - ptal.

**Pull request merged**